holidays/countries/cook_islands.py

```diff
@@ -0,0 +1,170 @@
+        * [Public Holidays (Aitutaki Gospel Day) Order 1989](https://web.archive.org/web/20250530121249/https://parliament.gov.ck/wp-content/uploads/2022/07/Public-Holidays-Aitutaki-Gospel-Day-Order-1990-No.-9.pdf)
+        * [Public Holidays Act 1999 (Consolidated Law)](https://web.archive.org/web/20250530101809/https://cookislandslaws.gov.ck/#/InternalConsolidatedLaws?legalId=LOCI.PHA99&parent_id=Public%20Holidays%20Act%201999)
+        * [Public Holidays (Ra o te Ui Ariki) Amendment Act 2011](https://web.archive.org/web/20250530101809/https://cookislandslaws.gov.ck/#/Laws-as-Made?id=1553&name=Public%20Holidays%20(Ra%20o%20te%20Ui%20Ariki)%20Amendment%20Act%202011%20No%204)
+        * [Public Holidays (Ra o te Ui Ariki) Amendment 2013](https://web.archive.org/web/20250530122050/https://parliament.gov.ck/wp-content/uploads/2020/02/Public-Holidays-Ra-o-te-Ui-Ariki-No.-4.pdf)
+        * [2004](https://web.archive.org/web/20090326004400/http://www.cook-islands.gov.ck/view_release.php?release_id=429)
+        * [2008](https://web.archive.org/web/20090326003301/http://www.stats.gov.ck/NewsEvents/PublicHolidays.pdf)
+    """
+
+    country = "CK"
+    default_language = "en_CK"
+    # %s (observed).
+    observed_label = tr("%s (observed)")
+    supported_languages = ("en_CK", "en_US")
+    # Public Holiday Act 1999.
+    start_year = 2000
+    subdivisions = (
+        "AI",  # Aitutaki (including uninhabited Manuae)
+        "AT",  # Atiu (including uninhabited Takutea)
+        "MG",  # Mangaia
+        "MH",  # Manihiki
+        "MT",  # Mitiaro
+        "PA",  # Palmerston
+        "PE",  # Penrhyn
+        "PU",  # Pukapuka (including Nassau and Suwarrow)
+        "RA",  # Rakahanga
+    )
+    subdivisions_aliases = {
+        "Aitutaki": "AI",
+        "Atiu": "AT",
+        "Mangaia": "MG",
+        "Manihiki": "MH",
+        "Mitiaro": "MT",
+        "Palmerston": "PA",
+        "Penrhyn": "PE",
+        "Pukapuka": "PU",
+        "Rakahanga": "RA",
+    }
+
+    def __init__(self, *args, **kwargs):
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        kwargs.setdefault("observed_rule", SAT_SUN_TO_NEXT_MON)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_observed(self._add_new_years_day("New Year's Day"), rule=SAT_SUN_TO_NEXT_MON_TUE)
+        self._add_observed(
+            self._add_new_years_day_two("Day after New Year's Day"), rule=SAT_SUN_TO_NEXT_MON_TUE
+        )
+
+        # Anzac Day.
+        self._add_anzac_day(tr("Anzac Day"))
+
+        # Good Friday.
+        self._add_good_friday(tr("Good Friday"))
+
+        # Easter Monday.
+        self._add_easter_monday(tr("Easter Monday"))
+
+        # Sovereign's Birthday.
+        self._add_holiday_1st_mon_of_jun(tr("Sovereign's Birthday"))
+
+        if self._year >= 2011:
```

Comment:
```suggestion
        if self._year >= 2012:
```
Public Holidays Amendment Act 2011 came into force on Jan 1, 2012.